CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Support for adding mock packages, for use where something is a dependency and you don't need it, or you need only a limited subset of the package. This is done using `micropip.add_mock_package`, `micropip.remove_mock_package` and `micropip.list_mock_packages`. Packages installed like this will be skipped by dependency resolution when you later install real packages.
 
+### Fixed
+
+- Prioritization of wheels by Python version in their tags: for example,
```

Comment:
```suggestion
- When multiple compatible builds for a package exist, the best build
  is now installed, as determined by the order of tags in  [`packaging.tags.sys_tags`](https://packaging.pypa.io/en/latest/tags.html#packaging.tags.sys_tags). For example,
```
as it's more general than Python version. It should also address pure python wheels vs compiled sources I think for instance.